src/Orleans.Core/Configuration/Options/MessagingOptions.cs

```diff
@@ -10,9 +10,22 @@ public abstract class MessagingOptions
     {
         /// <summary>
         /// The ResponseTimeout attribute specifies the default timeout before a request is assumed to have failed.
+        ///<seealso cref="ResponseTimeoutWithDebugger"/>
         /// </summary>
-        public TimeSpan ResponseTimeout { get; set; } = DEFAULT_RESPONSE_TIMEOUT;
-        public static readonly TimeSpan DEFAULT_RESPONSE_TIMEOUT = Debugger.IsAttached ? TimeSpan.FromMinutes(30) : TimeSpan.FromSeconds(30);
+        public TimeSpan ResponseTimeout
+        {
+            get { return Debugger.IsAttached ? RESPONSE_TIMEOUT_WITH_DEBUGGER : responseTimeout; }
```

Comment:
@benjaminpetit Shouldn't you use the `ResponseTimeoutWithDebugger` **property** instead of the `RESPONSE_TIMEOUT_WITH_DEBUGGER` **constant**?